src/lib/connection/connector/fallback-handlers.js

```diff
@@ -67,3 +67,13 @@ export function isIos() {
 export function openLink(url) {
   window.open(url, '_blank');
 }
+
+/**
+ * Open an external link
+ *
+ * @param {String} url address
+ */
+export function nativeUpload(url) {
+  // nothing we can do here
```

Comment:
maybe we should add a logging here

Reply:
there is an unmaterialized ticket for this